protostar-rust/tests/data/declare_test/src/lib.cairo

```diff
@@ -1,21 +1,28 @@
 mod contract1;
 
+#[starknet::interface]
+trait IHelloStarknet<TContractState> {
+    fn increase_balance(ref self: TContractState, amount: felt252);
+    fn decrease_balance(ref self: TContractState, amount: felt252);
+}
+
 #[starknet::contract]
 mod HelloStarknet {
     #[storage]
     struct Storage {
         balance: felt252,
     }
 
-    // Increases the balance by the given amount.
-    #[external]
-    fn increase_balance(ref self: ContractState, amount: felt252) {
-        self.balance.write(self.balance.read() + amount);
-    }
+    #[external(v0)]
```

Comment:
what means `v0`?

Reply:
https://community.starknet.io/t/cairo-1-contract-syntax-is-evolving/94794